### DISCUSSION TITLE
Welcome to cloud-atlas Discussions!

### DISCUSSION THREAD

**Discussion by huataihuang** [huataihuang/cloud-atlas#2] [General]
## 👋 Welcome!
  
所有和开源技术相关的问题，都欢迎你提出，我们一起讨论，一起学习:
  * 如果你对我撰写的文章有疑问或者发现我的错谬.
  * 如果你有新的建议.
  * 如果你有新的探索邀请我一起参与.
  * 欢迎一切开放的技术讨论.

如果你想开始，请在下面comment介绍你自己并告诉我们你想讨论的内容
  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by darkfriday** [comment]
這是本好書，無意中發現到的瑰寶。
希望您能繼續創作補全剩下內容。

**Reply by huataihuang** [reply]
> 這是本好書，無意中發現到的瑰寶。
> 希望您能繼續創作補全剩下內容。

谢谢鼓励，有时候想要写的文章，开了头还没有写完就要忙其他事情，确实自己也很惭愧。

我会持续实践和不断补充完整。如果您有建议或者疑问，请不吝赐教，我非常希望知道我撰写的文章中还存在哪些不足或没有解析清晰的地方，也希望知道阅读者对哪些方面更感兴趣。

我有很多想要学习和实践的方面，特别是自己每天看到还有很多不了解的知识和看到有很多前辈高手分享，更有动力加紧学习和分享。

感谢
